packages/core/src/components/forms/numericInputUtils.ts

```diff
@@ -28,18 +47,50 @@ export function getValueOrEmptyValue(value: number | string = "") {
     return value.toString();
 }
 
+function transformLocalizedNumberToStringNumber(character: string, locale: string) {
+    const charactersMap = new Array(10).fill("").map((_value, index) => index.toLocaleString(locale));
```

Comment:
could you also add a code comment here explaining this bit so that it's easier to comprehend at a glance? it's a bit roundabout... also it's not really a "map", is it?

```suggestion
    // ex. ['0', '1', '2', ..., '9']
    const characters = new Array(10).fill("").map((_value, index) => index.toLocaleString(locale));
```

Reply:
I guess I'll use a static array since it will never change